code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -236,19 +236,20 @@
 /obj/item/reagent_containers/food/drinks/mug/coco
 	name = "Dutch hot coco"
 	desc = "Made in Space South America."
-	list_reagents = list("hot_coco" = 30, "sugar" = 5)
+	list_reagents = list("hot_coco" = 15, "sugar" = 5)
 	foodtype = SUGAR
-
 	resistance_flags = FREEZE_PROOF
+	custom_price = 42
 
 
 /obj/item/reagent_containers/food/drinks/dry_ramen
 	name = "cup ramen"
 	desc = "Just add 10ml of water, self heats! A taste that reminds you of your school years."
```

Comment:
Absolutely anal, but this should be 5ml of water now that you've halved the ramen content.

Reply:
yeah that's fair